app/controllers/registration_tables_controller.rb

```diff
@@ -3,6 +3,8 @@ class RegistrationTablesController < ApplicationController
   before_action :get_event, :get_session, :get_table, :get_registration_tables, :get_possible_players
 
 
+  PREMIUM_MESSAGE = "This is a premium table. If payment isn't received within 1 hour, we will have to remove you from this table."
```

Comment:
I'd reword this part to something like: "This is a premium table, requiring payment. If payment is not received within an hour, you will be removed from this table". Or remove this message, as it is redundant with the message provided in the registration_tables html.